ton_client/src/encoding.rs

```diff
@@ -90,14 +90,14 @@ fn encode_base64(
     test: bool,
     as_url: bool,
 ) -> ClientResult<String> {
-    if let MsgAddressInt::AddrStd(address) = address {
+    if let MsgAddressInt::AddrStd(_) = address {
         let mut tag = if bounceable { 0x11 } else { 0x51 };
         if test {
             tag |= 0x80
         };
         let mut vec = vec![tag];
-        vec.extend_from_slice(&address.workchain_id.to_be_bytes());
-        vec.append(&mut address.address.get_bytestring(0));
+        vec.extend_from_slice(&address.workchain_id().to_be_bytes());
```

Comment:
This is not equal result. std workchain is `i8`. But here you use common `i32` workchain. So result will differ. If you want to modify this function you have to make unit test for it

Reply:
We do not merge any code changes from feature branches to master. This PR must be merged to a bugfix release branch 1.9.1-rc.